.github/workflows/ci.yml

```diff
@@ -45,9 +63,11 @@ jobs:
                 python-version: ${{ matrix.python-version }}
         -   name: "Main Script"
             run: |
-                EXTRA_INSTALL="numpy"
-                curl -L -O https://gitlab.tiker.net/inducer/ci-support/raw/main/build-and-test-py-project.sh
-                . ./build-and-test-py-project.sh
+                curl -L -O https://tiker.net/ci-support-v0
```

Comment:
Is this missing an `EXTRA_INSTALL=numpy`?

Reply:
`numpy` is an explicit dependency in `pyproject.toml`, so it should be installed.